config/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+module.exports = {
+    entry: './src/app/app.module.js',
+    output: {
+        filename: 'bundle.js',
+        path: path.resolve(__dirname, '../dist')
+    },
+    plugins: [new HtmlWebpackPlugin()],
```

Comment:
I think that you need to configure the pluging, at least adding the filename(resulting html) and template(your main html route) options